python_examples/simulation_flow.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pyEpiabm as pe
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
```

Comment:
Add some comments to illustrate the steps in the procedure to run the simulation. It will read better.